src/main/java/database/Database.java

```diff
@@ -0,0 +1,49 @@
+package main.java.database;
+
+import main.java.model.Notification;
+import main.java.model.User;
+
+public class Database {
+    private static Database instance;
+    private User currentUser;
```

Comment:
In addition to storing the current user's object, I think it would be a good idea to store their ID as well. This way we can minimize the number of requests to the database and clean up the rest of the codebase a bit.